compose/ui/ui/src/uikitMain/res/values-be/strings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2022 The Android Open Source Project
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+<resources xmlns:android="http://schemas.android.com/apk/res/android" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="next_page">Наступная старонка</string>
+    <string name="previous_page">Папярэдняя старонка</string>
+    <string name="first_page">Першая старонка</string>
+    <string name="last_page">Астатняя старонка</string>
```

Comment:
```suggestion
    <string name="last_page">Апошняя старонка</string>
```

Reply:
Anyways there is no belarusian lang in VoiceOver 😄

Reply:
Future hopes :)